udata/templates/search.html

```diff
@@ -52,11 +52,11 @@
             </a>
         </li>
     {% endif %}
-    {% if territory %}
+    {% if territories %}
         <li>
-            <a href="{{ url_for('territories.territory', territory=territory) }}">
-                {{ _('Territory:') }}
-                <span class="badge">{{ territory.name }}</span>
+            <a href="#territories" data-toggle="tab">
```

Comment:
OK, I will mix the two solution: 1 result, the tab redirects to the territory page, 2 and more, we list results.